doc/source/catalog_generation.rst

```diff
@@ -0,0 +1,361 @@
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+A flag value of 8 is assigned to sources whose signal to noise ratio is below a predefined value. We define sources as
+being above the faint object limit if the following is true:
+
+.. math::
+    \Delta ABmag_{outer} \leq  \frac{2.5}{snr \cdot log(10))}
+
+Where
+    * :math:`{\Delta ABmag_{outer}}` is the outer aperture AB magnitude uncertainty
+    * :math:`{snr}` is the signal to noise ratio, which is 1.5 for ACS/WFC and 5.0 for all other detectors.
+
+2.3.2.4: Assignment of flag value 32 (false detection: swarm around saturated source)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+The source identification routine has been shown to falsely identify sources in regions near bright or saturated
+sources, and in image artifacts associated with bright or saturated sources, such as diffraction spikes, and in the
+pixels surrounding saturated PSF where the brightness level “plateaus” at saturation. We identify impacted sources by
+locating all sources within a predefined radius of a given source and checking if the brightness of each of these
+surrounding sources is less than a radially-dependent minimum brightness value defined by a pre-defined stepped
+encircled energy curve. The parameters used to determine assignment of this flag are instrument-dependent, can be found
+in the “swarm filter” section of the \*_quality_control_all.json files in the path described above in section 1.3.
+
+
+2.3.2.5: Assignment of flag value 64 (False detection due proximity of source to image edge or other region with a low number of input images)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+Sources flagged with a value of 64 are flagged as “bad” because they are inside of or in close proximity to regions
+characterized by low or null input image contribution. These are areas where for some reason or another, very few or no
+input images contributed to the pixel value(s) in the drizzle-combined image.
+We identify sources impacted with this effect by creating a two-dimensional weight image that maps the number of
+contributing exposures for every pixel. We then check each source against this map to ensure that all sources and flag
+appropriately.
+
+3: The output catalog file
+---------------------------
+3.1: Filename format
+^^^^^^^^^^^^^^^^^^^^^^
+Source positions and photometric information are written to a .ecsv (Enhanced Character Separated Values) file. The
+naming of this file is fully automatic and follows the following format:
+<TELESCOPE>_<PROPOSAL ID>_<VISIT NUMBER>_<INSTRUEMENT>_<DETECTOR>_
+<FILTER>_<DATASET NAME>_<CATALOG TYPE>.ecsv
+
+So, for example if we have the following information:
+    * Telescope = HST
+    * Proposal ID = 98765
+    * Visit Number = 43
+    * Instrument = acs
```

Comment:
Same comment as above for "visit" vs "obset".

Reply:
"Visit number" changed to "Observation set ID" in commit [2e21c00](https://github.com/spacetelescope/drizzlepac/pull/760/commits/2e21c0043d31e857b7478b34e33362fc56392baf)